webpack.config.js

```diff
@@ -27,6 +28,9 @@ module.exports = {
     filename: "bundle.js",
     path: path.resolve(__dirname, "dist")
   },
+  plugins: [
+    new BundleAnalyzerPlugin({generateStatsFile: true})
+  ],
```

Comment:
This is how it's done in the examples
```
    new BundleAnalyzerPlugin({
      analyzerMode: process.env.ANALYZE_MODE || 'disabled',
    }),
```
In `package.json` scripts
```
 "analyze": "export ANALYZE_MODE=static && yarn build",
 ```
Not sure you need the stats file.